homeassistant/components/rest/sensor.py

```diff
@@ -267,44 +275,55 @@ class RestData:
     """Class for handling the data retrieval."""
 
     def __init__(
-        self, method, resource, auth, headers, data, verify_ssl, timeout=DEFAULT_TIMEOUT
+        self,
+        hass,
+        method,
+        resource,
+        auth,
+        headers,
+        data,
+        verify_ssl,
+        timeout=DEFAULT_TIMEOUT,
     ):
         """Initialize the data object."""
+        self._hass = hass
```

Comment:
Where is this used?

Reply:
It is not anymore (now that I replaced `__del__` with `async_remove`).

Will adjust

Reply:
fixed